src/main/java/HomeWork7/ToList.java

```diff
@@ -0,0 +1,29 @@
+package HomeWork7;
+
+import java.util.ArrayList;
```

Comment:
It's more than we need. Methods for transforming T[] to collection exists in many utils classes